image/L0ImageSmoothing/driver.py

```diff
@@ -0,0 +1,64 @@
+import argparse
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.